### PR TITLE
Alphabetically ordered CSS properties.

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

This PR will Alphabetically order the CSS properties and add a package Stylelint for automating the same.
(Resolves the Codacy issues regarding the order of CSS properties )
Currently, the changes are done using the npm package, so good review and discussion are needed before moving forward.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/LitmusStepper/styles.ts [hunk 1]
```diff
@@ -51,16 +51,16 @@ const useStyles = makeStyles((theme) => ({
...
 const useStepIconStyles = makeStyles((theme) => ({
   root: {
-    display: 'flex',
     alignItems: 'center',
-    justifyContent: 'center',
-    width: '1.5rem',
-    height: '1.5rem',
+    background: theme.palette.text.hint,
     borderRadius: '50%',
+    color: theme.palette.text.primary,
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/LitmusStepper/styles.ts:57]: This doesn't seem readable to me...previously all the display properties and flex related properties were together and width and height were together...sorting it alphabetically makes it less readable for me. For example, after sorting alignItems comes before display flex, this won't cause any CSS error and CSS will work just fine...but to a developer, it makes less sense I believe.

**Review by arkajyotiMukherjee — COMMENTED** [review]
This problem is replicated everywhere...I just pointed out a single instance.

File: litmus-portal/frontend/src/components/LitmusStepper/styles.ts (hunk 1 above)
Comment (Jonsy13) [reply litmus-portal/frontend/src/components/LitmusStepper/styles.ts:57]: Sure @arkajyotiMukherjee, I agree too.
No issues, this was just I case we want to enforce Alphabetically order for CSS properties.

**Comment by Jonsy13** [comment]
Closing it, as it is not required for now.

**Pull request closed without merging**